frb_codegen/src/library/utils/test_utils.rs

```diff
@@ -28,28 +28,43 @@ pub(crate) fn json_golden_test(
     let actual: Value = serde_json::from_str(&actual_str)?;
     debug!("json_golden_test sanitizers={sanitizers:?} actual:\n{actual_str}");
 
-    raw_golden_test(actual, &actual_str, matcher_path, |x| {
-        Ok(serde_json::from_str(&x)?)
-    })
+    raw_golden_test(
+        actual,
+        &actual_str,
+        matcher_path,
+        |x| Ok(serde_json::from_str(&x)?),
+        None,
```

Comment:
Not very sure, so here we do not assert anything?

Reply:
I'll refactor this to make it more obvious, but a default asserter is provided for None.

Reply:
Get it, no worries (just a quick nit)